git-and-github.md

```diff
@@ -51,6 +51,48 @@ every effort to divide it into smaller pieces.
 
 You as the person opening the pull request should assign a reviewer.
 
+### Pull Request with unreleased external dependencies
+
+Consider we have project A depending on project B. It's a common scenario that we
+are making changes to project B that will require project A to adapt to it (a "breaking change").
+While the changes to project B may still be under review and therefore unreleased, it can be
+valuable to open a PR against project A to show the migration path for these upcoming changes.
+
+For us to be able to already use the new, unreleased version, we can set the dependency version of
+package B to a temporary tag, commit or branch on GitHub. This ensures the PR can do a proper CI
+run against the upcoming changes, allowing it to reveal potential problems that otherwise may
+go unnoticed.
+
+We can use the following syntax to specify a dependency directly from GitHub without creating an
+actual package for it.
+
+```
+"package-name@git+https://github.com/owner/repository.git@tag"
+```
+
+Let's assume package B is called `lahja` and the `setup.py` file normally specifies the version in
+the `install_requires` or `extras_requires` section as follows.
+
+```
+...
```

Comment:
bonus quotes?

Reply:
One of us might not be looking at this the right way. These are three dots (`...`) indicating that these are samples out of a longer list.